app/models/ai_persona.rb

```diff
@@ -98,6 +98,8 @@ def class_instance
     mentionable = self.mentionable
     default_llm = self.default_llm
     max_context_posts = self.max_context_posts
+    vision_enabled = self.vision_enabled
+    vision_max_pixels = self.vision_max_pixels
```

Comment:
Oh, we should also have a model validator catch big values for this, something like:

```ruby
validates :vision_max_pixels, numericality: { greater_than: 0, maximum: 1_048_576 }
```

Reply:
added